ipld/kamt/src/iter.rs

```diff
@@ -0,0 +1,154 @@
+// Copyright 2021-2023 Protocol Labs
+// SPDX-License-Identifier: Apache-2.0, MIT
+use std::borrow::Borrow;
+use std::iter::FusedIterator;
+
+use fvm_ipld_blockstore::Blockstore;
+use fvm_ipld_encoding::de::DeserializeOwned;
+use fvm_ipld_encoding::CborStore;
+
+use crate::hash_bits::HashBits;
+use crate::node::{match_extension, ExtensionMatch, Node};
+use crate::pointer::Pointer;
+use crate::{AsHashedKey, Config, Error, KeyValuePair};
+#[doc(hidden)]
+pub struct IterImpl<'a, BS, V, K, H, const N: usize = 32> {
```

Comment:
Let's rename this to `Iter` and remove the `#[doc(hidden)]`. We only did it that way because we had multiple versions.

Reply:
done